MAUI/LinearGauge/overview.md

```diff
@@ -0,0 +1,28 @@
+---
+layout: post
+title: About .NET MAUI Linear Gauge control | Syncfusion
+description: Learn here all about introduction of Syncfusion .NET MAUI Linear Gauge (SfLinearGauge) control, its features, and more.
+platform: maui
+control: SfLinearGauge
+documentation: ug
+---
+
+# .NET MAUI Linear Gauge (SfLinearGauge) Overview
+
+Syncfusion .NET MAUI Linear Gauge is a data visualization control to display data on a linear scale. Use this control to craft high-quality mobile app user interfaces.
+
+![Overview .NET MAUI linear gauge](images/basic_elements.PNG)
+
+## Key Features
+
+* **Orientation** - The Linear Gauge can be set to vertical or horizontal orientation.
+* **Scale** - The scale is a set of values can be plotted. An scale can be customized by changing the thickness and edge styles and you can also inverse the scale.
+* **Labels and Ticks** - The axis elements such as labels, major ticks, and minor ticks can be customized to different styles.
+* **Ranges** - A range is a visual element that helps you to quickly visualize where a range falls on the scale track. Multiple ranges with different styles can be added to the Linear Gauge.
+* **Pointers** - A pointer is used to indicate a specific value on an scale. The gauge has three types of pointers: shape marker pointer, content marker pointer, and bar pointer. All the pointers can be customized as needed and you can also add multiple pointers in the Linear Gauge.
+* **Mirror Gauge** - The Linear Gauge can be mirrored. When the [`IsMirrored`]() property is true, all the gauge elements will be rendered in mirror effect. 
+* **Animation** - Animate the gauge elements in a visually appealing way, when they are load, or when their values are changed.
```

Comment:
Animate the gauge elements in a visually appealing manner when they are loaded or their values are changed.